### PR TITLE
DO NOT LAND [Gradio Styles]

### PR DESCRIPTION
**Pull request by zakariassoul**
DO NOT LAND [Gradio Styles]

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/810).
* #813
* #812
* __->__ #810
* #777

### CODE REVIEW

File: python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx [hunk 1]
```diff
@@ -23,8 +23,8 @@ export default memo(function PromptName({ promptId, name, onUpdate }: Props) {
   return (
     <TextInput
       value={nameInput}
+      label="Prompt Name"
```
Comment (rholinshead) [inline python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx:26]: You want the labels in both gradio and local editor?

**Review by rholinshead — COMMENTED** [review]
Some minor things, but PMd as well since we'll need the code changes to land (with local editor styles where applicable) and just have gradio-only styles as "do not land"

File: python/src/aiconfig/editor/client/src/components/ParametersRenderer.tsx [hunk 2]
```diff
@@ -105,20 +105,20 @@ const ParameterInput = memo(function ParameterInput(props: {
...
           autosize
           size="xs"
+          minRows={3}
           maxRows={5}
           onChange={(event) => {
             setParameterValueString(event.target.value);
             debouncedCellParameterUpdate(parameterName, event.target.value);
           }}
         />
+        <ActionIcon
```
[the remark this answers is not in the document]
Comment (zakariassoul) [reply python/src/aiconfig/editor/client/src/components/ParametersRenderer.tsx:115]: Yeah. The button is below in the design. It still needs different styling, but ok for now.

File: python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx (hunk 1 above)
Comment (zakariassoul) [reply python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx:26]: i use the mantine class to force label display: "none" in local-editor.

**Pull request closed without merging**